examples/tests/NavBarSearchAppearance.ttl

```diff
@@ -12,60 +12,60 @@ stage:NavbarSearchAppearance a elmo:Part;
 		elmo:data [
 			elmo:index "1";
 			rdfs:label "Appearances";
-			elmo:data [elmo:index "01"; rdfs:label "HtmlAppearance"; html:link "/"];

```

Comment:
It might be better to change this. Creating two seperate definitions (one at root and one at ldt) seems illogical. But we might do something else:
- If the link starts with "/", it is absolute and starts from the ROOT;
- If the link doesn't starts with "/", it is relative to the stage (docroot and possible domain).

This we can change in de navbarappearance itself.